src/work-allocation/components/task-field/task-field.component.html

```diff
@@ -32,4 +32,7 @@
   <ng-container *ngSwitchCase="fieldType.CASE_REFERENCE">
     <exui-case-reference-field [caseReference]="task[config.name]"></exui-case-reference-field>
   </ng-container>
+  <ng-container *ngSwitchCase="fieldType.DERIVED_ICON">
+    <exui-derived-icon-field [task]="task" [sourceColumn]="sourceColumn" [matchValue]="matchValue"></exui-derived-icon-field>
```

Comment:
Should pass `[config]="config"` instead.

Reply:
Ah, you either need change the `DerivedIconFieldComponent` to take a `config` input or these should be passed from `config.sourceColumn`, etc.